packit_service/models.py

```diff
@@ -204,7 +204,7 @@ def get_all(cls) -> Optional[Iterable["CoprBuild"]]:
 
     # Returns all builds with that build_id, irrespective of target
     @classmethod
-    def get_all_build_id(
+    def get_all_by_build_id(
```

Comment:
How it is possible to get multiple builds for one `build_id`?

Reply:
because in copr, one build_id is built in different chroots (terrible schema)
we have a line in DB for every chroot (better schema)

Reply:
Thanks for the clarification.

Reply:
Thanks for the clarification.